multy_core/src/bitcoin/bitcoin_account.cpp

```
@@ -297,5 +297,45 @@ AccountPtr make_bitcoin_account(const char* private_key)
     return AccountPtr(new BitcoinAccount(std::move(key), HDPath(), net_type));
 }
 
+BinaryDataPtr parse_bitcoin_address(const char* address,
+                                    BitcoinNetType* net_type,
+                                    enum BitcoinAddressType* address_type)
+{
+    BinaryDataPtr out_binary_data;
+    size_t binary_data_size = 0;
+    THROW_IF_WALLY_ERROR(
+                wally_base58_get_length(address, &binary_data_size),
+                "Can not get the len");
+    make_binary_data(binary_data_size, reset_sp(out_binary_data));
```

1. Error is not handled here.
2. Use vector for decoded data.